logstash-core-event/spec/logstash/event_spec.rb

```diff
@@ -44,6 +44,11 @@
         subject["@metadata"] = { "action" => "index" }
         expect(subject["[@metadata][action]"]).to eq("index")
       end
+
+      it "should add key when setting nil value" do
```

Comment:
Mild curiosity - both core-event-java and core-event have this test, but the implementation is not the same, though it's really close. Should this use a shared example (sometime soon in a later PR?)